mne/defaults.py

```diff
@@ -10,25 +10,26 @@
     color=dict(mag='darkblue', grad='b', eeg='k', eog='k', ecg='m',
                emg='k', ref_meg='steelblue', misc='k', stim='k',
                resp='k', chpi='k', exci='k', ias='k', syst='k',
-               seeg='k', dipole='k', gof='k', bio='k'),
+               seeg='k', dipole='k', gof='k', bio='k', ecog='k'),
```

Comment:
I feel like we could be more funky with our colors in the future. At least we could have a color scheme where the data channels would be easier to separate.
In following PRs...